packages/components/src/guide/page-control.js

```diff
@@ -18,7 +18,10 @@ export default function PageControl( { currentPage, numberOfPages, setCurrentPag
 	return (
 		<ul className="components-guide__page-control" aria-label={ __( 'Guide controls' ) }>
 			{ times( numberOfPages, ( page ) => (
-				<li key={ page }>
+				<li
+					key={ page }
+					aria-current={ page === currentPage }
```

Comment:
For my own curiosity: Would one of the keywords associated with `aria-current` have been appropriate to use here? Notably, I'm wondering if `'step'` could have been used as an indication of the active step in the set of guide steps?

```jsx
aria-current={ page === currentPage ? 'step' : undefined }
```

>A `step` token used to indicate a link within a step indicator for a step-based process, where the link is visually styled to represent the current step.

https://www.w3.org/TR/wai-aria-1.1/#aria-current

Reply:
Thank you, yes, this is even better. I filed #19290 to change this.